doc/lsst.daf.relation/sql.rst

```diff
@@ -15,10 +15,10 @@ Multi-engine relation trees can be executed by a `.Processor` subclass.
 This engine flattens back-to-back `.Join` and `.Chain` operations, and it actually reorders any combination of adjacent `.Join`, `.Selection`, `.Calculation`, and `.Projection` into a single ``SELECT...FROM...WHERE`` statement via the commutation rules for these operations.
 It assumes the database query optimizer will reorder at least these operations itself anyway, so the goal is to keep the query as simple as possible to stay out of its way and aid human readers.
 
-The `Engine.to_executable` method transforms a `.Relation` tree to a SQL ``SELECT`` or ``UNION`` thereof, as represented by `SQLAlchemy_`.
+The `Engine.to_executable` method transforms a `.Relation` tree to a SQL ``SELECT`` or ``UNION`` thereof, as represented by `SQLAlchemy`_.
 The engine's payload type is the `Payload` `~dataclasses.dataclass`, which better maps to SQL tables or very simple subqueries that can be used like tables.
 
-Support for custom `.UnaryOperation` subclasses can be added implementing `Engine.apply_custom_unary_operation`.
+Support for custom `.UnaryOperation` subclasses can be added implementing `~lsst.daf.relation.iteration.Engine.apply_custom_unary_operation`.
```

Comment:
This:
```suggestion
Support for custom `.UnaryOperation` subclasses can be added implementing `~lsst.daf.relation.sql.Engine.apply_custom_unary_operation`.
```
is the correct full path (there are different `Engine` subclasses in each engine subpackage), but I think the original should have worked due to the `py:currentmodule::` directive at the top of the file.

Reply:
Hmm. It was complaining that it couldn't find it. Will try again or at least fix the bug.

Reply:
I remember now. There is no method `apply_custom_unary_operation` on the sql `Engine` class. That's why it can't find out. It seems like the sentence should be tweaked a bit.

Reply:
Ah, sorry about that.  Dug a little deeper, and this is what it should say:
```suggestion
Support for custom `.UnaryOperation` subclasses can be added by reimplementing `Engine.to_payload` while delegating to `super` for the standard operation classes.
```